marlowe-integration/changelog.d/20230620_075519_brian.bush_PLT_5817.rst

```diff
@@ -0,0 +1,4 @@
+Added
+-----
+
+- Fifteen-second timeout for contract safety analysis.
```

Comment:
We are not doing changelog tracking in this folder

Reply:
Fixed in 34f1d7e5d89e68996254f165386a3264372c3252.